pyccel/codegen/printing/fcode.py

```diff
@@ -2242,6 +2250,11 @@ def _print_MathPow(self, expr):
         base_c = self._print(base)
         e_c    = self._print(e)
         return '{} ** {}'.format(base_c, e_c)
+    def _print_NumpyFabs(self, expr):
+        arg = expr.args[0]
+        code_args = self._print(arg)
+        code = 'fabs({})'.format(code_args)
+        return self._get_statement(code)
```

Comment:
Why can't this function be handled by `_print_NumpyUfuncBase`? The main difference that I see is that the argument is not converted to a real. Is this unnecessary for fabs and necessary for almost every other NumpyUfuncBase or have I missed something?

Reply:
Same question here...

Reply:
yes, you are right, we don't need it. I add it just for testing purposes. I forgot to delete it.
maybe it is the problem of different return types I got. I will make sure of it.